diff -ur gajim-0.9.org/src/common/connection.py gajim-0.9/src/common/connection.py
--- gajim-0.9.org/src/common/connection.py	2005-12-23 14:20:58.000000000 +0100
+++ gajim-0.9/src/common/connection.py	2005-12-24 00:51:16.000000000 +0100
@@ -1287,7 +1287,7 @@
 			try:
 				j = helpers.parse_jid(jid)
 			except:
-				print >> sys.sterr, _('Jid %s is not RFC compliant. It will not be added to your roster. Use roster management tools such as http://jru.jabberstudio.org/ to remove it') % jid
+				print >> sys.stderr, _('Jid %s is not RFC compliant. It will not be added to your roster. Use roster management tools such as http://jru.jabberstudio.org/ to remove it') % jid
 			else:
 				roster[j] = raw_roster[jid]
 
