diff -urN gajim-0.8.orig/Makefile gajim-0.8/Makefile
--- gajim-0.8.orig/Makefile	2005-08-19 12:28:20.699646352 +0200
+++ gajim-0.8/Makefile	2005-08-19 12:30:10.959884248 +0200
@@ -3,10 +3,10 @@
 GAJIM_AP	= 0 # do we build Autopackage?
 
 MODULES		= src src/common po
-PREFIX		= /usr/local
+PREFIX		= /usr
 DESTDIR		= 
 LIBDIR		= /lib
-MANDIR		= $(DESTDIR)$(PREFIX)/share/man
+MANDIR		= $(PREFIX)/share/man
 
 FIND		= find . \( -name '*.glade' -o -name '*.py' -o -name '*.xpm' -o -name '*.gif' -o -name '*.png' -o -name '*.mo' -o -name '*.wav' \)
 
@@ -73,9 +73,9 @@
 	cp data/pixmaps/gajim_about.png "$(DESTDIR)$(PREFIX)/share/pixmaps/";
 	mkdir -p "$(DESTDIR)$(PREFIX)/share/applications";
 	cp gajim.desktop "$(DESTDIR)$(PREFIX)/share/applications/";
-	mkdir -p "$(MANDIR)/man1";
-	cp gajim.1 "$(MANDIR)/man1";
-	cp gajim-remote.1 "$(MANDIR)/man1";
+	mkdir -p "$(DESTDIR)$(MANDIR)/man1";
+	cp gajim.1 "$(DESTDIR)$(MANDIR)/man1";
+	cp gajim-remote.1 "$(DESTDIR)$(MANDIR)/man1";
 	mkdir -p "$(DESTDIR)$(PREFIX)$(LIBDIR)/gajim";
 	for f in $(FILES_LIB) ; do \
 		cp "$$f" "$(DESTDIR)$(PREFIX)$(LIBDIR)/gajim/"; \
