diff -urN gajim-0.8.1.orig/scripts/gajim-remote.py gajim-0.8.1/scripts/gajim-remote.py
--- gajim-0.8.1.orig/scripts/gajim-remote.py	2005-09-04 11:21:28.417658008 +0200
+++ gajim-0.8.1/scripts/gajim-remote.py	2005-09-04 11:45:01.257873648 +0200
@@ -211,7 +211,7 @@
 		except:
 			self.send_error(_('Session bus is not available.'))
 		
-		if _version[1] >= 30 and _version[1] <= 42:
+		if _version[1] >= 30:
 			obj = self.sbus.get_object(SERVICE, OBJ_PATH)
 			interface = dbus.Interface(obj, INTERFACE)
 		elif _version[1] < 30:
