diff -Nurwa gajim-0.13.4/src/common/passwords.py gajim-0.13.4.keyring-bug//src/common/passwords.py
--- gajim-0.13.4/src/common/passwords.py	2010-01-20 10:59:33.000000000 +0100
+++ gajim-0.13.4.keyring-bug//src/common/passwords.py	2010-10-27 12:02:16.784214246 +0200
@@ -63,7 +63,7 @@
 	def __init__(self):
 		self.keyring = gnomekeyring.get_default_keyring_sync()
 		if self.keyring is None:
-			self.keyring = 'default'
+			self.keyring = 'login'
 		try:
 			gnomekeyring.create_sync(self.keyring, None)
 		except gnomekeyring.AlreadyExistsError:
