diff -urN gajim-0.8.1.orig/Makefile gajim-0.8.1/Makefile
--- gajim-0.8.1.orig/Makefile	2005-09-04 12:48:06.972357568 +0200
+++ gajim-0.8.1/Makefile	2005-09-04 13:08:35.816544856 +0200
@@ -64,6 +64,7 @@
 			mkdir -p "$(DESTDIR)$(PREFIX)/share/gajim/$$d"; \
 		fi; \
 	done
+	mv scripts/gajim-remote.py src
 	for f in $(FILES) ; do \
 		DST=`dirname "$$f"`; \
 		cp "$$f" "$(DESTDIR)$(PREFIX)/share/gajim/$$DST/"; \
@@ -92,6 +93,7 @@
 		echo "$$F" > "$(DESTDIR)$(PREFIX)/bin/$$BASE"; \
 		chmod +x "$(DESTDIR)$(PREFIX)/bin/$$BASE"; \
 	done
+	rmdir "$(DESTDIR)$(PREFIX)/share/gajim/scripts"
 
 #
 # show make params we accept
diff -urN gajim-0.8.1.orig/scripts/gajim-remote gajim-0.8.1/scripts/gajim-remote
--- gajim-0.8.1.orig/scripts/gajim-remote	2005-09-04 12:48:06.975357112 +0200
+++ gajim-0.8.1/scripts/gajim-remote	2005-09-04 12:56:51.448625056 +0200
@@ -22,6 +22,6 @@
 	echo "You must not launch gajim-remote as root, it is INSECURE"
 fi
 
-cd PREFIX/share/gajim/scripts
+cd PREFIX/share/gajim/src
 export PYTHONPATH="$PYTHONPATH:PREFIXLIB/gajim"
 python gajim-remote.py $@
diff -urN gajim-0.8.1.orig/scripts/gajim-remote.py gajim-0.8.1/scripts/gajim-remote.py
--- gajim-0.8.1.orig/scripts/gajim-remote.py	2005-09-04 12:48:06.976356960 +0200
+++ gajim-0.8.1/scripts/gajim-remote.py	2005-09-04 12:56:38.362614432 +0200
@@ -32,7 +32,7 @@
 
 signal.signal(signal.SIGINT, signal.SIG_DFL) # ^C exits the application
 
-import i18n
+from common import i18n
 
 _ = i18n._
 i18n.init()
