--- gajim-0.14.4/src/common/passwords.py~	2010-10-01 22:19:07.000000000 +0200
+++ gajim-0.14.4/src/common/passwords.py	2011-07-29 22:20:09.067102579 +0200
@@ -63,7 +63,7 @@
     def __init__(self):
         self.keyring = gnomekeyring.get_default_keyring_sync()
         if self.keyring is None:
-            self.keyring = 'default'
+            self.keyring = 'login'
         try:
             gnomekeyring.create_sync(self.keyring, None)
         except gnomekeyring.AlreadyExistsError:
